assets/js/modules/analytics/datastore/profiles.js

```diff
@@ -162,8 +164,8 @@ const baseSelectors = {
 	 * @param {string} propertyID The Analytics Property ID to check for profile creation.
 	 * @return {boolean} `true` if creating a profile, `false` if not.
 	 */
-	isDoingCreateProfile: createRegistrySelector( ( select ) => ( state, propertyID ) => {
-		return select( STORE_NAME ).isFetchingCreateProfile( propertyID );
+	isDoingCreateProfile: createRegistrySelector( ( select ) => ( state, propertyID, profileName ) => {
+		return select( STORE_NAME ).isFetchingCreateProfile( propertyID, profileName );
```

Comment:
I understand why this was added here but I feel like the name isn't really important for this selector.
Unfortunately, it is needed to use the underlying `isFetching` selector. What we can do though instead is change this selector to return true if _any_ create profile request is in progress (we have a few other selectors which do a similar thing). 

E.g.
https://github.com/google/site-kit-wp/blob/5efc53aad0542751420381cbf8b522d10b5d5b50/assets/js/modules/analytics/datastore/accounts.js#L328-L334

Let's do the same here, since we could always technically select `isFetchingCreateProfile` if really needed.

Reply:
Updated.